src/static/js/animal_new.js

```diff
@@ -578,6 +580,18 @@ $(function() {
                 $("#animalname").val(response); 
             });
 
+            $("#species").change(function() {
+                let speciesid = $("#species").val();
+                $.each($(".additional"), function(i, af) {
+                    if ($(af).attr("data-speciesids").split(",").includes(speciesid)) {
+                        $(af).closest("tr").show();
```

Comment:
this is ok for now, but closest("tr") is going to fail one day when we want to get rid of using tables for layouts and switch to bootstrap. I will add a separate ticket for refactoring and having tableform output some kind of form field/row class with all form field rows to use instead